CovertInfiltration/Config/XComStrategyTuning.ini

```diff
@@ -151,10 +151,60 @@ PrototypePrimaries=true
 PrototypeSecondaries=true
 PrototypeArmorsets=true
 
-COST_MEDIKIT_MK1=15
-COST_MEDIKIT_MK2=25
-COST_FLASHGRENADE=13
-COST_SMOKEGRENADE=10
+; Change the cost of hardcoded vanilla items
+
++arrItemCostOverrides=(ItemName="Medikit", NewCost=(ResourceCosts[0]=(ItemTemplateName="Supplies", Quantity=15)), Difficulties=(0,1,2))
```

Comment:
I think it will be easier to read if it was `ItemName` and then `Difficulties` (and the `NewCost` last)